### PR TITLE
minidsp: init at 0.1.9

### PR DESCRIPTION
**Pull request by adamcstephens**
###### Description of changes

https://github.com/mrene/minidsp-rs

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/minidsp/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    lib.optionals stdenv.isLinux [libusb1]
+    ++ lib.optionals stdenv.isDarwin [AppKit IOKit];
+
+  nativeBuildInputs = lib.optionals stdenv.isLinux [pkg-config];
+
+  meta = with lib; {
+    description = "A control interface for some MiniDSP products";
+    homepage = "https://github.com/mrene/minidsp-rs";
+    license = licenses.asl20;
+    platforms = platforms.linux ++ platforms.darwin;
```
Comment (sikmir) [inline pkgs/applications/audio/minidsp/default.nix:37]:
```suggestion
    platforms = platforms.unix;
```

Comment (adamcstephens) [reply pkgs/applications/audio/minidsp/default.nix:37]: This is straight from the upstream flake. I make no claims or expectations that the package will work on the other platforms in `platforms.unix`.

Comment (mrene) [reply pkgs/applications/audio/minidsp/default.nix:37]: the dependency on `hidapi` will fail to build outside of linux/darwin/windows - the app could be built on these systems 
 (given the right cargo features) but would only work with networked devices.  

feel free to add me as a maintainer as well :)

**Review by mrene — APPROVED** [review]
Result of `nixpkgs-review pr 221696` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>minidsp</li>
  </ul>
</details>

**Pull request merged**